moveit_ros/planning/planning_request_adapter_plugins/res/default_plan_request_adapter_params.yaml

```diff
@@ -0,0 +1,44 @@
+default_plan_request_adapter_parameters:
+  path_tolerance: {
+    type: double,
+    description: "AddTimeOptimalParameterization: Tolerance per joint in which the time parameterization is allowed to deviate from the original path.",
+    default_value: 0.1,
+  }
+  resample_dt: {
+    type: double,
+    description: "AddTimeOptimalParameterization: Time steps between two waypoints of the parameterized trajectory. The trajectory is re-sampled from the original path.",
+    default_value: 0.1,
+  }
+  min_angle_change: {
+    type: double,
+    description: "AddTimeOptimalParameterization: Minimum joint value change to consider two waypoints unique.",
+    default_value: 0.001,
+  }
+  start_state_max_dt: {
+    type: double,
+    description: "FixStartStateCollision/FixStartStateBounds: Maximum temporal distance of the fixed start state from the original state.",
+    default_value: 0.5,
+  }
+  jiggle_fraction: {
+    type: double,
+    description: "FixStartStateCollision: Joint value perturbation as a percentage of the total range of motion for the joint.",
+    default_value: 0.02,
+  }
+  max_sampling_attempts: {
+    type: int,
+    description: "FixStartStateCollision: Maximum number of attempts to re-sample a valid start state.",
+    default_value: 100,
+    validation: {
+        bounds<>: [0, 255]
```

Comment:
Why the upper bound here? Maybe I want to sample all day... :smile:

Reply:
:+1: 
![upper_bound_gone](https://github.com/ros-planning/moveit2/assets/44898064/807c3e29-b635-48d6-af82-0d2a52eb1dfc)

Reply:
in that case, does the adapter take care of timeouts? If not, unlimited sampling wouldn't be too useful